### PR TITLE
Switch drop-in envs to work with uwsgi server with one worker

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
**### This will be merged after drum release and prod drop in envs update.**

Enable uwsgi based prediction server with one worker in drop in envs.
For now, these envs will be deployed only on staging.

--with-error-server param has been removed. Error server has been used to serve error messages, if model loading fails.
With uwsgi server, loading fails on worker, so main drum process won't fail. Instead worker will be serving error message.

### CODE REVIEW

File: dev_env/java_nginx_env/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # This is the default base image for use with user models and workflows.
 FROM datarobot/java-dropin-env-base
 
-RUN apt-get update && apt-get install -y vim nginx
-
-RUN chmod 707 /var/lib/nginx
+RUN apt-get update && apt-get install -y vim
```
Comment (meltzg) [inline dev_env/java_nginx_env/Dockerfile:4]: why do we install vim?

Comment (yakov-g) [reply dev_env/java_nginx_env/Dockerfile:4]: These are development/test/debug envs, so just to have it there.

File: public_dropin_environments/r_lang/start_server.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 cd /opt/code/ || exit 1
 export PYTHONPATH=/opt/code
 
-CMD="drum server -cd . --address 0.0.0.0:8080 --with-error-server"
+CMD="drum server -cd . --address 0.0.0.0:8080 --production --max-workers 1 --show-stacktrace"
```
Comment (meltzg) [inline public_dropin_environments/r_lang/start_server.sh:5]: do we no longer need the error server?

**Comment by yakov-g** [comment]
> environment version IDs need to be updated

Envs ID's have been updated before and installed on prod.
So envs are uninstallable on staging. Do you want to update them once again to differ from prod.?

File: public_dropin_environments/r_lang/start_server.sh (hunk 2 above)
Comment (yakov-g) [reply public_dropin_environments/r_lang/start_server.sh:5]: --with-error-server param has been removed. Error server has been used to serve error messages, if model loading fails.
With uwsgi server, loading fails on worker, so main drum process won't fail. Instead worker will be serving error message.

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**6.23** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**